docs/user_manual/calculations.md

```diff
@@ -8,25 +8,83 @@ SPDX-License-Identifier: MPL-2.0
+- [State estimation](#state-estimation-algorithms): a statistical method that calculates the most probabilistic state of the grid, given sensor values with an uncertainty. This calculation can be performed by using the {py:class}`calculate_state_estimation <power_grid_model.PowerGridModel.calculate_state_estimation>` method. An example of usage of the power-flow calculation function is given in [State Estimation Example](../examples/State%20Estimation%20Example.ipynb)
 
 ### Calculation types explained
-TODO: 
-- What is the difference between power flow and state estimation
-- When should you use which? Maybe small example (physical, not code)
+
+#### Power flow
+Power flow is a "what-if" based grid calculation that will calculate the node voltages and the power flow through the branches, based on assumed load/generation profiles.
+Some typical use-cases are network planning and contingency analysis.
+
+Input:
+- Network data: topology + component attributes
+- Assumed load/generation profile
+
+Output:
+- Node voltage magnitude and angle
+- Power flow through branches
+
+#### State estimation
+State estimation is a statistical calculation method that determines the most probable state of the grid, based on
+network data and measurements. Measurements meaning power flow or voltage values with some kind of uncertainty, which were 
+either measured, estimated or forecasted.
+
+Input:
+- Network data: topology + component attributes
+- Power flow / voltage measurements with uncertainty
+
+Output:
+- Node voltage magnitude and angle
+- Power flow through branches
+- Deviation between measurement values and estimated state
+
+In order to perform a state estimation the system should be observable. If the system is not observable the calculation will 
+raise a singular matrix error. Simply said, observability means that the number of measurements
+should be larger than or equal to the number of unknowns. For each node there are two unknowns, `u` and `u_angle`, so the following
+equations should be met:
+
+$$
+   \begin{eqnarray}
+      n_{measurements}    & >= & n_{unknowns}
+   \end{eqnarray}
+$$
+
+Where
+
+$$
+   \begin{eqnarray}
+      n_{unknowns}    & = & 2 & \cdot & n_{nodes}
+   \end{eqnarray}
+$$
+
+The number of measurements can be found by the sum of the following:
+- number of nodes with a voltage sensor with magnitude only
+- two times the number of nodes with a voltage sensor with magnitude and angle
+- two times the number of nodes without appliances connected
+- two times the number of nodes where all connected appliances are measured by a power sensor
+- two times the number of branches with a power sensor
+
+Note: enough measurements doesn't necessarily mean that the system is observable. The location of the measurements is also
```

Comment:
Should we include a simple drawing of a net where it is obvious that the location of the measurements is important?